Dockerfile

```diff
@@ -1,13 +1,13 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-FROM fedora@sha256:5ce8497aeea599bf6b54ab3979133923d82aaa4f6ca5ced1812611b197c79eb0
+FROM fedora@sha256:d0207dbb078ee261852590b9a8f1ab1f8320547be79a2f39af9f3d23db33735e
 
 # --- Version Pinning --- #
 
 ARG fedora="40"
 ARG arch="x86_64"
 
-ARG version_csdiff="3.4.0-1"
+ARG version_csdiff="3.5.0-1"
```

Comment:
Note that `csdiff-3.5.1` is just behind the door: https://bodhi.fedoraproject.org/updates/FEDORA-2024-39c7b47798

Reply:
I saw it; thanks for the feedback. I'll update to 3.5.1 once it is available in stable.